other-governance-configs/acala-governance.rs

```diff
@@ -0,0 +1,247 @@
+parameter_types! {
```

Comment:
This `other-governance-configs` folder is for reference of Karura/Moonriver/Khala configurations.
The files only include the relevant pallets.
Will remove this when we are done.